tests/testthat/test-build-home-index.R

```diff
@@ -25,3 +25,12 @@ test_that("package repo verification", {
   )
 })
 
+test_that("page title",{
+  pkg <- test_path("assets/pagetitle")
+  expect_equal(as.character(data_home(pkg))[[1]],
+               "Such a cool package")
+
+  pkg <- test_path("assets/home-index-rmd")
```

Comment:
I think this should be in a separate test called something like "page title defaults to package title", and it should come before the test for the override.